tests/core/test_abort_on_exception_post.js

```diff
@@ -1,46 +1,46 @@
-addOnPostRun(function() {

```

Comment:
To prevent line ending inconsistencies, the following should be added to .gitattributes. This will prevent the new files to change the line-ending. 
```
* text=auto eol=lf
*.{cmd,[cC][mM][dD]} text eol=crlf
*.{bat,[bB][aA][tT]} text eol=crlf
```

Reply:
https://github.com/emscripten-core/emscripten/pull/12980/commits/cfc730c03d95706026c73c988de13ddf366f61e0